app/forms/teacher_interface/delete_file_form.rb

```diff
@@ -0,0 +1,8 @@
+class TeacherInterface::DeleteFileForm
```

Comment:
```suggestion
class TeacherInterface::DeleteUploadForm
```

Similar comment with the naming of the other form.